lib/bootloader_pvm.pm

```diff
@@ -179,7 +179,11 @@ sub boot_hmc_pvm {
     # don't wait for it, otherwise we miss the menu
     type_string "mkvterm -m $hmc_machine_name --id $lpar_id\n";
     # skip further preperations if system is already installed
-    return if get_var('BOOT_HDD_IMAGE');
+    # if cannot find grub2 screen, send up key to refresh serial output
```

Comment:
Hi Yutao,

I think we can add more comments here, like:

Sometimes, guest boots up very quickly, it is already in grub2 boot menu entry and waiting for the next operation before connecting to the serial terminal. so we add some workaround here to send a "up" key to refresh the terminal